### PR TITLE
feat(policies): add Shipping card with handling time to Policies page

### LINKED ISSUE #278: feat: add Shipping Policies card with handling time to Policies page
## Summary

Add a **Shipping** card to the AI Storefront Policies settings page that lets merchants specify their order handling time (min/max business days). This data is emitted into product JSON-LD as `ShippingDeliveryTime` on the `OfferShippingDetails` block, making it machine-readable for AI crawlers and search engines.

## Schema.org context

`OfferShippingDetails` supports a `deliveryTime` property → `ShippingDeliveryTime`, which in turn has:

```json
"deliveryTime": {
  "@type": "ShippingDeliveryTime",
  "handlingTime": {
    "@type": "QuantitativeValue",
    "minValue": 1,
    "maxValue": 3,
    "unitCode": "DAY"
  }
}
```

`transitTime` can also be added in a future iteration but is harder to derive without carrier data — keeping scope tight.

### Settings (PHP)
- Two new settings keys: `wc_ai_storefront_handling_time_min` and `wc_ai_storefront_handling_time_max`
- Sanitized as non-negative integers; `max >= min` validation
- Stored via the existing `WC_AI_Storefront_Settings` class pattern

### Policies page (JS)
- New **Shipping** card alongside the existing Return Policy card
- Two numeric inputs: "Min handling time (days)" / "Max handling time (days)"
- Only emitted in JSON-LD when both values are set and valid

### JSON-LD emitter (PHP)
- `add_shipping_details()` enriched with `deliveryTime` block when handling time settings are present
- `ShippingDeliveryTime.handlingTime` with `minValue`, `maxValue`, `unitCode: "DAY"`

### Tests
- PHP unit tests: `handlingTime` block emitted when both settings present; omitted when either is missing; `maxValue < minValue` edge case
- JS: settings save/load (follows existing Policies page test patterns)

## Acceptance criteria

- [ ] Min/max handling time inputs appear on the Policies page under a "Shipping" heading
- [ ] Values are saved and restored correctly
- [ ] Product JSON-LD includes `deliveryTime.handlingTime` when both values are set
- [ ] JSON-LD omits `deliveryTime` when either value is absent or invalid
- [ ] PHP unit tests cover all three scenarios above
- [ ] No regression on existing return policy card or `shippingRate` emission (issue #277)

## Related

- #277 — `shippingRate: 0` emission for free shipping (sibling feature on `OfferShippingDetails`)

### PR DESCRIPTION
**Pull request by pierorocca**
## Summary

- New **Shipping** card on the Policies page with min/max business-day stepper inputs for order handling time
- Values are sanitized (`WC_AI_Storefront_Handling_Time`) and stored as `handling_time: { min, max }` in plugin settings
- JSON-LD emitter enriches `OfferShippingDetails.deliveryTime` with a `ShippingDeliveryTime.handlingTime` QuantitativeValue when both `min > 0` and `max > 0`; omits the block when either is 0 (unconfigured)
- `StepperInput` widget extracted from the return-window stepper and reused for both fields
- `deriveHandlingTimePreview()` exported from `policies-tab.js` mirrors the PHP emitter contract for test parity
- min/max cross-clamping: setting min above current max raises max; setting max below current min lowers min

## Test plan

- [ ] 5 PHP unit tests: emit when both set; omit when min=0; omit when max=0; omit when setting absent; omit when no shippingDetails block
- [ ] 5 JS unit tests: null when min=0; null when max=0; null for empty; QuantitativeValue when both positive; same value for min===max
- [ ] `composer test` passes (1104 tests)
- [ ] `npm run test:js` passes (143 tests)
- [ ] No regression on return policy card or existing `shippingDestination` / `shippingRate` emission

Closes #278

### CODE REVIEW

File: includes/ai-storefront/class-wc-ai-storefront-handling-time.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Handling-time sanitizer.
+ *
+ * Stores the merchant's declared order handling time (min/max business days)
+ * so the JSON-LD emitter can output a `ShippingDeliveryTime.handlingTime`
+ * QuantitativeValue on the `OfferShippingDetails` block.
+ *
+ * @package WooCommerce_AI_Storefront
+ * @since 0.9.0
```
[the remark this answers is not in the document]
Comment (pierorocca) [reply includes/ai-storefront/class-wc-ai-storefront-handling-time.php]: Removed — the `@since 0.9.0` tag was premature since that version hasn't been released yet.
